dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -18,7 +18,7 @@
 		<slf4j.version>2.0.6</slf4j.version>
 
         <opentelemetry.version>1.35.0</opentelemetry.version>
-        <mptelemetry.version>2.0-RC1</mptelemetry.version>
```

Comment:
[This part of the tck suite](https://github.com/OpenLiberty/open-liberty/blob/856b566b0d56e1f979516db2e5cec36ab379e23f/dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/tck/tck-suite-application.xml#L23-L29) needs removing now